src/scenes/bootloader.js

```diff
@@ -0,0 +1,78 @@
+export default class Bootloader extends Phaser.Scene {
+  constructor() {
+    super({ key: "bootloader" });
+  }
+
+  preload() {
+    this.createBars();
+    this.setLoadEvents();
+    this.loadSprites();
+    this.loadAudios();
+    this.loadMaps();
+  }
+
```

Comment:
**praise**: Good use of separate methods for loading different types of assets, which enhances readability and maintainability.